### ISSUE TITLE
Fans always spinning

### ISSUE THREAD

**Issue by proudh0n** [rugarciap/Turbo-Boost-Switcher#47]
When I'm using Turbo Boost Switcher, after a couple minutes the fans start spinning at min RPM and they won't stop until I reset the SMC. This seems to happen with both turboboost enabled and disabled.

As far as I know having the fans always spinning is not something desirable, and it doesn't make any sense either. So I thought it must be a bug.

MacBook Pro 14,2 (13" mid 2017 with Touchbar)
Turbo Boost Switcher 2.6.1
macOS High Sierra 10.13.2

**Comment by rugarciap** [comment]
Hi!

Thanks for your feedback.

Turbo Boost Switcher doesn't mess around with the fans at all (yet ;)..., it "just" enables / disables a flag on MSR record for Turbo Boost to be enabled /disabled as documented. Indirectly, fans usually lower down its speed when turbo boost is disable, and that's what happens mostly when disabling turbo boost and doing intensive computing tasks.

Take a look at other apps running and/or use an app like smcFanControl or so to control your fans speed.

Closing this issue.

Best.

**Comment by proudh0n** [comment]
Hi, thank you for answering. I'm really new at Macs and I don't really know how things work but until I removed Turbo Boost Switcher the fans didn't stop spinning... I've resetted the SMC like 3 times and after booting, with TBS installed, the fans always always started spinning at min RPM with only 35º.

I don't want to open this issue again, so keep it closed. Right now I've installed TBS again and it's fine. Maybe was some kind of strange interaction and it was a coincidence that removing TBS resolved the issue.

Thank you for your attention!
